modules/op-chains-signing-secrets-ecdsa.adoc

```diff
@@ -0,0 +1,53 @@
+// This module is included in the following assemblies:
+// * secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="chains-signing-secrets-ecdsa_{context}"]
+= Generating the x509 key pair
+
+You can generate the `x509` key pair by setting the `generateSigningSecret` parameter to `true`. 
+The {pipelines-title} Operator generates an `ecdsa` type key pair: `x509.pem` private key and `x509-pub.pem` public key, and stores it in the `signing-secrets` secret in the `openshift-pipelines` namespace.
+
+The {pipelines-title} Operator *does not* provide the following functions to limit potential security issues:
+
+* Key rotation 
+* Auditing key usage
+* Proper access control to the key
+
+[IMPORTANT]
+====
+Ensure that you store the `x509-pub.pem` public key in a safe place to verify later artifact attestations.
+====
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`) utility.
+* You are logged in to your {OCP} cluster with administrative rights for the `tekton-chains` namespace.
+
+.Procedure
+
+. Edit the `TektonConfig` CR by running the following command:
++
+[source,terminal]
+----
+$ oc edit TektonConfig config
+----
+
+. In the `TektonConfig` CR, set the `generateSigningSecret` value to `true`:
++
+.Example of creating ecdsa key pair
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+# ...
+  chain:
+    disabled: false
+    generateSigningSecret: true # <1>
+# ...
+----
+<1> The default value is `false`. Setting the value to `true` generates the `ecdsa` key pair.
```

Comment:
Yeah @jkhelil can you please add the section regarding the behavior for first point

And for the second point I agree with what you have  said

Reply:
by default, it is false, there is an empty signing-secret (or assume the user has created its signing-secret), 
- we moved to true:  it  updates the signing-secrets secret in  openshift-pipelines (updates because there is an already empty secret)
- we moved to false: it overrides the signing-secrets secret in openshift-pipelines and emptied it
its is important to say
The user should retrieve and store in a safe place the x509-pub.pem public key to verify later artifact attestations, when using this field, the x509-pub.pem is part of  signing-secrets secret in openshift-pipelines